src/components/about/About.module.css

```
@@ -2,12 +2,11 @@
   display: flex;
   flex-wrap: wrap;
   column-gap: 100px;
-  padding: 50px;
 }
 
 .aboutTitle {
-  margin-top: 50px;
-  margin-bottom: 60px;
+  margin: 0;
+  padding: 60px 0;
```

I would say, this is a margin, not a padding, bu it is up to you
```suggestion
  margin: 60px 0;
```

---

Fixed

---

Fixed